editor/src/clj/editor/workspace.clj

```diff
@@ -446,19 +452,40 @@ ordinary paths."
     (if (.exists dir)
       (delete-directory-recursive dir))))
 
+(def ^:private bin-zip-names
+  (into #{"/plugins/bin/common.zip"}
+        (comp
+          (mapcat (juxt #(.getPair ^Platform %) #(.getOs ^Platform %)))
+          (map #(str "/plugins/bin/" % ".zip")))
+        [Platform/X86_64Linux Platform/X86_64MacOS Platform/X86_64Win32]))
+
+(defn- bin-zip? [resource]
+  (let [path (resource/proj-path resource)]
+    (boolean (some #(string/ends-with? path %) bin-zip-names))))
+
+(defn- unpack-bin-zip! [workspace resource]
+  (let [proj-path (resource/proj-path resource)
+        plugin-file (plugin-path workspace proj-path)
+        infix-path (subs proj-path 0 (- (count proj-path) 4))]
```

Comment:
Might want to assert that `proj-path` ends in `".zip"` here?

Reply:
Yep, fixed